<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>771</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>771</width>
      <height>311</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>身份证号码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>航班号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>航班日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>里程(/km)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>600</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Adobe Devanagari&quot;;
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>vip客户：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="vip_table">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>640</y>
      <width>541</width>
      <height>141</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 11, 64);
background-color:white;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>身份证号码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>乘机频率</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>总里程数(/km)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="file_open_buttom">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>数据列表：</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>解决冲突策略：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="hash_method">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>线性探测法</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>再散列法</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>平方探测法</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>链地址法</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>HASH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>260</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(表长：200)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openfile"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="search_by_ID"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openfile">
   <property name="text">
    <string>选择文件</string>
   </property>
  </action>
  <action name="search_by_ID">
   <property name="text">
    <string>按身份证号查找</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
